botrc.example

```diff
@@ -35,3 +35,7 @@ export HUBOT_QUOTEFILE_PATH=
 # mapping
 
 export HUBOT_MAPPING_DATAROOT=botdata/
+
+# weather
+
+export HUBOT_WEATHER_APIKEY=
```

Comment:
Thanks for remembering to include this! Makes it a lot easier to remember what to include once I'm changing the deployment. :sparkles: